[\#499 Issue](https://github.com/rear/rear/issues/499) `closed`: Failed to start udev Kernel Device Manager (rear 1.16.1-git201410241343 )
==========================================================================================================================================

**Labels**: `waiting for info`, `support / question`

#### <img src="https://avatars.githubusercontent.com/u/9460796?v=4" width="50">[tomid](https://github.com/tomid) opened issue at [2014-10-30 09:59](https://github.com/rear/rear/issues/499):

I try to make a backup of my Arch-Linux-System (Kernek 3.17.1-1) via
ISO/NFS and recover it in a vSphere-VM (P2V). The backup runs well and
there are no errors in log. When i try to recover the system in virtual
machine, i get the error "Failed to start udev Kernel Device Manager"
several times, when i boot the ISO Image with "Automatic recover".
Therefore iÂ´ve no ethenet-device and an unreachable network, so REAR
cannnot access the Backup-Archiv on NFS. Anyone any idea? I could try to
backup on USB, but this is no the solution, i want. My
/etc/rear/local.conf:  
OUTPUT=ISO  
BACKUP=NETFS  
BACKUP\_URL="nfs://192.168.0.252/volume1/NAS"  
OUTPUT\_URL="nfs://192.168.0.252/volume1/NAS"  
ISO\_ISOLINUX\_BIN="/usr/lib/syslinux/bios/isolinux.bin"  
AUTOEXCLUDE\_PATH=/mnt  
AUTOEXCLUDE\_DISKS=y

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2014-10-30 12:06](https://github.com/rear/rear/issues/499#issuecomment-61081561):

@tomid do you have the same problem if you do the same in *manual* mode?

#### <img src="https://avatars.githubusercontent.com/u/9460796?v=4" width="50">[tomid](https://github.com/tomid) commented at [2014-10-30 16:44](https://github.com/rear/rear/issues/499#issuecomment-61125328):

@gdha, yes. There is no difference.

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2014-12-10 07:17](https://github.com/rear/rear/issues/499#issuecomment-66413109):

@tomid did your system detect a network driver (grep in dmesg output)?
Perhaps you are missing the proper driver for it? Or still need to load
it manually? If you are performing a P2V remember you have to make sure
are drivers required in vShere are present in the rescue image.

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2015-02-06 20:21](https://github.com/rear/rear/issues/499#issuecomment-73305712):

@tomid I remember a case (P2V situation) where an additional network was
required (which was not present in rear image). It was required to add
this kernel module in the `/etc/rear/local.conf` as:

    MODULES=( ${MODULES[@]} my_network_module )
    MODULES_LOAD=( ${MODULES_LOAD[@]} my_network_module )

when we ran `rear mkbackup`

#### <img src="https://avatars.githubusercontent.com/u/1788608?u=925fc54e2ce01551392622446ece427f51e2f0ce&v=4" width="50">[jsmeix](https://github.com/jsmeix) commented at [2016-09-21 13:10](https://github.com/rear/rear/issues/499#issuecomment-248607408):

No fedback since more than one year =&gt; closing it.

------------------------------------------------------------------------

\[Export of Github issue for
[rear/rear](https://github.com/rear/rear).\]
